app/assets/stylesheets/watch-listen.css.scss

```diff
@@ -0,0 +1,16 @@
+.watch-listen {
+  .video-wrapper {
+    height: 0;
+    overflow: hidden;
+    padding-bottom: 56.25%; // For ratio 16:9. 75% if ratio is 4:3
+    position: relative;
+    margin: (2 * $gutter) 0 (2 * $gutter) 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `2 * $gutter 0` instead of `2 * $gutter 0 2 * $gutter 0`